meteor/client/lib/forms/schemaFormTableArray.tsx

```diff
@@ -0,0 +1,254 @@
+import { faPlus, faSync } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { clone, objectPathGet, objectPathSet } from '@sofie-automation/corelib/dist/lib'
+import React, { useCallback, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { WrappedOverridableItemNormal, OverrideOpHelperForItemContents } from '../../ui/Settings/util/OverrideOpHelper'
+import { useToggleExpandHelper } from '../../ui/Settings/util/ToggleExpandedHelper'
+import { doModalDialog } from '../ModalDialog'
+import {
+	getSchemaSummaryFieldsForObject,
+	SchemaFormSofieEnumDefinition,
+	SchemaSummaryField,
+	translateStringIfHasNamespaces,
+} from './schemaFormUtil'
+import { JSONSchema } from '@sofie-automation/shared-lib/dist/lib/JSONSchemaTypes'
+import { getSchemaDefaultValues } from '@sofie-automation/shared-lib/dist/lib/JSONSchemaUtil'
+import { hasOpWithPath } from '../Components/util'
+import { SchemaTableSummaryRow, SchemaFormTableEditRow } from './schemaFormTableShared'
+import { literal } from '@sofie-automation/shared-lib/dist/lib/lib'
+import { SchemaFormUIField } from '../../../lib/jsonSchemaUtil'
+
+interface SchemaFormTableProps {
+	schema: JSONSchema
+	translationNamespaces: string[]
+	sofieEnumDefinitons?: Record<string, SchemaFormSofieEnumDefinition>
+
+	attr: string
+
+	item: WrappedOverridableItemNormal<any>
+	overrideHelper: OverrideOpHelperForItemContents
+}
```

Comment:
I think it would be good to add some documentation here to explain what these properties do.